releasePlan.cfg

```diff
@@ -11,5 +11,5 @@
 jdk8uGA="jdk8u392"
 jdk11uGA="jdk-11.0.21"
 jdk17uGA="jdk-17.0.9"
-jdk21GA="jdk-21"
+jdk21GA="jdk-21.0.1"
```

Comment:
Should be 21u ?

Reply:
If `jdk-21.0.1` is a tag reference, then that looks correct to me for October and JDK 21. It's expected to be a GA tag of `jdk-21.0.1-ga`.

Reply:
The string is just the expected version in tag format (without any build), as it is then used for input to tag sorting to determine if a "jdk-21.0.1-ga" exists yet.